.circleci/config.yml

```
@@ -47,14 +53,29 @@ jobs: # a basic unit of work in a run
       - run:
           name: Build and test
           command: |
-            . common.sh
-            source /opt/mambaforge/etc/profile.d/conda.sh
-            source /opt/mambaforge/etc/profile.d/mamba.sh
-            mamba activate bioconda
-            bioconda-utils build recipes config.yml \
-                --lint --docker --mulled-test \
-                --docker-base-image "quay.io/bioconda/bioconda-utils-build-env-cos7-$(arch):${BIOCONDA_UTILS_TAG#v}" \
-                --git-range origin/master HEAD
+            set -e
```

```suggestion
            set -xe
```
I like using `x` for printing the executed commands, not just their output. It helps for debugging.